golem-templates/templates/c/c-app-component-example-http/components-cpp/component-name/src/component.c

```diff
@@ -15,11 +15,11 @@ static uint64_t total = 0;
 
 // Implementation of the exported functions.
 // See component_name.h for the generated function signatures.
-void exports_pack_name_api_add(uint64_t value) {
+void exports_c_http_exports_c_http_api_add(uint64_t value) {
```

Comment:
did the package name went away? or any generated parts? i guess

Reply:
that's not intentional :) it had to be changed because of the "-exports" thing. but not to the hardcoded other parts